### PR TITLE
Fixes/v6/single worlds

### PR DESCRIPTION
**Pull request by dordsor21**
Fix single worlds

### CODE REVIEW

**Comment by N0tMyFaultOG** [comment]
I'm not very familiar with world plots, but setting `worlds: true` and trying to claim a world plot in a non-plot area told me I'm not in a plot area? But it created the following in the worlds.yml http://haste.athion.net/gorayupive.yml

**Comment by dordsor21** [comment]
> I'm not very familiar with world plots, but setting `worlds: true` and trying to claim a world plot in a non-plot area told me I'm not in a plot area? But it created the following in the worlds.yml http://haste.athion.net/gorayupive.yml
> 
> > **[hastebin](http://haste.athion.net/gorayupive.yml)**

"worlds" means the IDs are *;0;0 with the worlds (now being) named 0_0, -1_5, etc accordingly. Worlds is not and as never been "any" world.

**Comment by N0tMyFaultOG** [comment]
Oic http://haste.athion.net/okibebawoc.md

**Review by N0tMyFaultOG — APPROVED** [review]

File: Bukkit/src/main/java/com/plotsquared/bukkit/managers/BukkitWorldManager.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
 import org.bukkit.World;
 import org.bukkit.WorldCreator;
 import org.bukkit.WorldType;
+import org.jetbrains.annotations.NotNull;
```
Comment (Citymonstret) [inline Bukkit/src/main/java/com/plotsquared/bukkit/managers/BukkitWorldManager.java:35]: Wrong annotation

File: Core/src/main/java/com/plotsquared/core/plot/world/SinglePlotManager.java [hunk 2]
```diff
@@ -57,11 +57,11 @@ public SinglePlotManager(@Nonnull final PlotArea plotArea) {
     }
 
     @Override public Location getPlotBottomLocAbs(@Nonnull final PlotId plotId) {
-        return Location.at(plotId.toCommaSeparatedString(), -30000000, 0, -30000000);
+        return Location.at(plotId.toSeparatedString("_"), -30000000, 0, -30000000);
     }
 
     @Override public Location getPlotTopLocAbs(@Nonnull final PlotId plotId) {
-        return Location.at(plotId.toCommaSeparatedString(), 30000000, 0, 30000000);
+        return Location.at(plotId.toSeparatedString("_"), 30000000, 0, 30000000);
```
Comment (Citymonstret) [inline Core/src/main/java/com/plotsquared/core/plot/world/SinglePlotManager.java:64]: Add a `toUnderscoreSeparatedString()` to `PlotId`, as the behaviour is repeated so often. Also consider making `30000000` a constant (`MAX_COORDINATE` or whatever)

**Review by Citymonstret — COMMENTED** [review]
Really minor code style related stuff. These changes make sense and it's nice that single worlds have been given some love

**Review by Citymonstret — APPROVED** [review]

**Pull request merged**